clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -49,6 +50,16 @@ const ScanResultsForm = () => {
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
 
+  const { successAlert } = useAlert();
+
+  useEffect(() => {
+    successAlert(
```

Comment:
I didn't realize this was a thing—I'm guessing it's from fidesops? At a certain point, we should figure out how this jives with https://github.com/ethyca/fides/blob/238/scan-complete-alerts/clients/admin-ui/src/features/common/toast.tsx. We should make a 🎫 to do so

Reply:
made a ticket: https://github.com/ethyca/fides/issues/1696